l10n/en-US/viewer.properties

```diff
@@ -263,3 +263,6 @@ editor_stamp_add_image.title=Add image
 editor_free_text2_aria_label=Text Editor
 editor_ink2_aria_label=Draw Editor
 editor_ink_canvas_aria_label=User-created image
+
+# Alt-text dialog
```

Comment:
Please add to the comment what an "alt text" is (I would also stay consistent with the way is spelled in the string)